test/run-openshift

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# Test the httpd image in OpenShift.
+#
+# IMAGE_NAME specifies a name of the candidate image used for testing.
+# VERSION specifies a version of the python in the candidate image.
+# The image has to be available before this script is executed.
+
+THISDIR=$(dirname ${BASH_SOURCE[0]})
+
+source ${THISDIR}/test-lib.sh
+source ${THISDIR}/test-lib-openshift.sh
```

Comment:
These need to be double quoted.  Also `IMAGE_NAME` needs double quotes added when the tests are invoked.

Reply:
Good point, will fix.